### ISSUE TITLE
GIF files do not appear as GIF while Previewing the Post from Draft

### ISSUE THREAD

**Issue by atapas** [Hashnode/user-feedback#74]
**Describe the bug**
While Drafting a post if I upload a gif file and go to Preview, the gif file doesn't appear as gif. Rather it appears as a static image without any animations in it. I am facing this problem only from 12th May 2019. Before the behavior was as expected.

This gives me lower confidence in publishing my post as I am unsure if the GIF will appear correctly even after publishing. If not, the post will not communicate things well.

**To Reproduce**
1. Go to 'Write a Story' from the user menu of the blog.
2. Draft a new Story.
3. Upload a GIF.
4. Go to Preview Tab.
5. The GIF doesn't appear as GIF, it appears as a static image.

**Expected behavior**
GIF should appear as GIF with all the animation in it.

**Screenshots**
NA

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Mozilla Firefox and Chrome
 - Version [Chrome 74.03, FF 66]

**Smartphone (please complete the following information):**
 NA

**Additional context**
Tested with old GIFs that are working for already published posts. Facing same issue.

**Comment by sandeeppanda92** [comment]
Hey Tapas! We recently switched to [imgix](https://www.imgix.com/) for serving our images. It looks like they have some problem with animated gifs. So, we have reverted the switch temporarily till gif issue has been fixed. We've raised a ticket with them.

**TLDR:** You can go ahead and publish the post. The gifs will appear correctly both in preview and after being published.
